Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -5,33 +5,68 @@ import BraintreePayPal
 class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
 
     lazy var payPalClient = BTPayPalClient(apiClient: apiClient)
+    
+    lazy var emailLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Buyer email:"
+        return label
+    }()
+    
+    lazy var emailTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "placeholder@email.com"
+        textField.backgroundColor = .systemBackground
+        return textField
+    }()
+    
+    lazy var payLaterToggleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Offer Pay Later"
+        label.font = .preferredFont(forTextStyle: .footnote)
+        return label
+    }()
+    
+    let payLaterToggle = UISwitch()
 
     override func createPaymentButton() -> UIView {
         let payPalCheckoutButton = createButton(title: "PayPal Checkout", action: #selector(tappedPayPalCheckout))
         let payPalVaultButton = createButton(title: "PayPal Vault", action: #selector(tappedPayPalVault))
-        let payPalPayLaterButton = createButton(title: "PayPal with Pay Later Offered", action: #selector(tappedPayPalPayLater))
 
-        let buttons = [payPalCheckoutButton, payPalVaultButton, payPalPayLaterButton]
-        let stackView = UIStackView(arrangedSubviews: buttons)
+        let stackView = UIStackView(arrangedSubviews: [
+            UIStackView(arrangedSubviews: [emailLabel, emailTextField]),
+            buttonsStackView(label: "1-Time Checkout", views: [
+                UIStackView(arrangedSubviews: [payLaterToggleLabel, payLaterToggle]),
+                payPalCheckoutButton
+            ]),
+            buttonsStackView(label: "Vault",views: [payPalVaultButton])
+        ])
+        
         stackView.axis = .vertical
-        stackView.alignment = .center
-        stackView.distribution = .fillEqually
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 25
         stackView.translatesAutoresizingMaskIntoConstraints = false
-        
         return stackView
     }
+    
+    // MARK: - 1-Time Checkout Flows
 
     @objc func tappedPayPalCheckout(_ sender: UIButton) {
         progressBlock("Tapped PayPal - Checkout using BTPayPalClient")
         sender.setTitle("Processing...", for: .disabled)
         sender.isEnabled = false
 
         let request = BTPayPalCheckoutRequest(amount: "5.00")
+        request.userAuthenticationEmail = emailTextField.text
+        
         let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "5.00", name: "item one 1234567", kind: .debit)
         lineItem.upcCode = "123456789"
         lineItem.upcType = .UPC_A
         lineItem.imageURL = URL(string: "https://www.example.com/example.jpg")
         request.lineItems = [lineItem]
+        
+        if payLaterToggle.isOn {
+            request.offerPayLater = true
+        }
```

Comment:
Take it or leave it:
```suggestion
        request.offerPayLater = payLaterToggle.isOn ? true : false
```